rss_parser/models/atom/feed.py

```diff
@@ -0,0 +1,61 @@
+from typing import Optional
+
+from rss_parser.models import XMLBaseModel
+from rss_parser.models.atom.entry import Entry
+from rss_parser.models.image import Image
+from rss_parser.models.types.date import DateTimeOrStr
+from rss_parser.models.types.only_list import OnlyList
+from rss_parser.models.types.tag import Tag
+from rss_parser.pydantic_proxy import import_v1_pydantic
+
+pydantic = import_v1_pydantic()
+
+
+class RequiredAtomFeedMixin(XMLBaseModel):
+    feed_id: Tag[str] = pydantic.Field(alias="id")
```

Comment:
why not id? It's not shadowing the builtin id function afaik, since it's an attribute